content/en/docs/concepts/cluster-administration/logging.md

```diff
@@ -132,6 +132,29 @@ directory should be rotated. In Kubernetes clusters brought up by
 the `kube-up.sh` script, those logs are configured to be rotated by
 the `logrotate` tool daily or once the size exceeds 100MB.
 
+## Node Log Viewer
+
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+Kubernetes v1.25 offers initial support for viewing node logs using kubectl.
```

Comment:
```suggestion
Kubernetes v{{< skew currentVersion >}} includes initial support for viewing node logs using `kubectl`.
```

That way it's right for each subsequent release. The feature-state shortcode tells readers what version provides this support (and if that's not clear, we should improve the shortcode).